packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -23,9 +23,9 @@ Okta supports many ways of authenticating the identity of a user during the sign
 
 The Android and iOS Identity Engine SDKs represent the sign-in flow as a state machine. You initialize the machine with the details of your Okta org app integration, request the initial step in the flow, and cycle through responding to steps until either the user signs in, cancels, or an error occurs.
 
-<div class="common-image-format">
+<div class="common-image-format bordered-image">
 
-![](/img/mobile-sdk/mobile-idx-basic-flow.png "A diagram that shows the sign-in flow.")
+!["A diagram that shows the sign-in flow."](/img/mobile-sdk/mobile-idx-basic-flow.png)
```

Comment:
```suggestion
!["A diagram that shows the mobile app processes in a sign-in flow."](/img/mobile-sdk/mobile-idx-basic-flow.png)
```
Or you can use "mobile app states in your code for the sign-in flow".  This diagram is specific to the process steps or states in the app code for the sign-in flow.

Reply:
fixed